### PR TITLE
Chapter 4: Rails-flavored Ruby

### PR DESCRIPTION
**Pull request by nhattien2307**

### CODE REVIEW

**Comment by nhattien2307** [comment]
screenshot from 2019-02-15 08-18-29

**Comment by nhattien2307** [comment]
ready

File: app/views/static_pages/help.html.erb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<% provide(:title, "Help") %>
+<h1>Help</h1>
```
Comment (chien-pv) [inline app/views/static_pages/help.html.erb]: I18n nhé em

**Comment by chien-pv** [comment]
commented
3 commit rồi gộp lại thành 1, fix conflicts luôn

**Comment by nhattien2307** [comment]
ready

**Comment by chien-pv** [comment]
merged

**Pull request merged**